src/components/views/rooms/EventTile.js

```diff
@@ -98,6 +98,9 @@ module.exports = React.createClass({
 
         /* a function to be called when the highlight is clicked */
         onHighlightClick: React.PropTypes.func,
+
+        /* is this the focussed event */
+        selectedEvent: React.PropTypes.bool,
```

Comment:
Nit: I would prefer if this were named `isSelectedEvent` so it is obvious that it is a `boolean`, else it could be mistaken for a `MatrixEvent`.